cmd/admin-router.go

```diff
@@ -53,6 +53,9 @@ func registerAdminRouter(router *mux.Router, enableIAM bool) {
 	// Info operations
 	adminV1Router.Methods(http.MethodGet).Path("/info").HandlerFunc(httpTraceAll(adminAPI.ServerInfoHandler))
 
+	// Performance command - retrun performance details based on input type
+	adminV1Router.Methods(http.MethodGet).Path("/performance").HandlerFunc(httpTraceAll(adminAPI.PerfInfoHandler)).Queries("perfType", "{perfType:.*}")
```

Comment:
This should be moved into XL only @nitisht as @kannappanr mentioned

Reply:
> This should be moved into XL only

But we may want to return disk perf for FS deployments too. How about adding a check in `PerfInfoHandler`?

Reply:
There is no value in running for FS mode IMO - this is generally to centrally run across many disks. FS mode one can simply run `dd`

Reply:
okay, updated. @harshavardhana @kannappanr PTAL